pycbc/workflow/grb_utils.py

```diff
@@ -365,3 +400,177 @@ def get_sky_grid_scale(
         if upscale:
             out *= rayleigh.interval(2 * containment - 1)[-1]
     return out
+
+
+def setup_pygrb_pp_workflow(wf, pp_dir, seg_dir, segment, insp_files,
+                            inj_files, inj_insp_files, inj_tags):
+    """
+    Generate post-processing section of PyGRB offline workflow
+    """
+    pp_outs = FileList([])
+    # Begin setting up trig combiner job(s)
+    # Select executable class and initialize
+    exe_class = select_grb_pp_class(wf, "trig_combiner")
+    job_instance = exe_class(wf.cp, "trig_combiner")
+    # Create node for coherent no injections jobs
+    node, trig_files = job_instance.create_node(wf.ifos, seg_dir, segment,
+                                    insp_files, pp_dir)
+    wf.add_node(node)
+    pp_outs.append(trig_files)
+
+    # Trig clustering for each trig file
+    exe_class = select_grb_pp_class(wf, "trig_cluster")
+    job_instance = exe_class(wf.cp, "trig_cluster")
+    for trig_file in trig_files:
+        # Create and add nodes
+        node, out_file = job_instance.create_node(trig_file, pp_dir)
+        wf.add_node(node)
+        pp_outs.append(out_file)
+
+    # Find injections from triggers
+    exe_class = select_grb_pp_class(wf, "inj_finder")
+    job_instance = exe_class(wf.cp, "inj_finder")
+    inj_find_files = FileList([])
+    for inj_tag in inj_tags:
+        tag_inj_files = FileList([f for f in inj_files
+                                  if inj_tag in f.tags])
+        tag_insp_files = FileList([f for f in inj_insp_files
+                                   if inj_tag in f.tags[1]])
```

Comment:
Is the `[1]` index intended on the last line (and not intended a couple of lines before)? If so, consider adding a comment explaining why you only look at the second element.

Reply:
I think this was because the first file in the inspiral file list is (but is not always) the bank veto bank. Looking at it now, I can probably come up with a better way to get around it.

Reply:
Any update on this?

Reply:
Please check this or at least add a FIXME comment so we do not forget.